### DISCUSSION TITLE
Shortcuts

### DISCUSSION THREAD

**Discussion by AnonBlankzy** [Slion/Fulguris#646] [Ideas]
Styx Browser was able to have shortcuts on the homepage. If Fulguris did this & added cookie exceptions it would be the best browser out there in my opinion

**Comment by Slion** [comment]
Can't you use the bookmarks as home page?

**Reply by AnonBlankzy** [reply]
Yes I can do that however I prefer to have an option to add shortcuts instead. The reason for that is I have a lot of bookmarks, most of which I don't use all the time. What's great about shortcuts on the home screen is that you can put a few of your most used websites there and don't have to search through all the bookmarks. Styxx Browser (based on fulguris) had a shortcut option although you could only add 4, it was better than none imo. Duckduckgo browser does great at this as well as Firefox and Vivaldi. If Fulguris ever gets shortcuts and the option to  delete all cookies on exit except for the ones you want to keep, I'd never use another browser to be honest.

**Reply by Slion** [reply]
Just put your bookmarks in a folder. Put your shortcuts at the root. Done.
